### ISSUE TITLE
Settings changes made via WP-CLI are not being logged

### ISSUE THREAD

**Issue by fjarrett** [xwp/stream#697]
For some reason the `get_called_class()` function inside the `WP_Stream_Connector::callback()` method is returning a bogus result when invoking any `$ wp option` command.

**Example**

```bash
$ wp option update blogdescription "Foo"
```

The `get_called_class()` value returns `WP_Stream_Connector_Widgets` when it should be returning `WP_Stream_Connector_Settings`. This result is that no record is ever created because the correct callback is never called.

All other Connectors appear to be logging activity just fine when using WP-CLI, only the Settings connector seems to be affected.

**Comment by fjarrett** [comment]
Update: It seems that the `get_current_class()` value is _always_ returning as `WP_Stream_Connector_Widgets` when using WP-CLI.

**Comment by fjarrett** [comment]
**Update:** After further testing I discovered that every connector _is_ logging activity when using WP-CLI commands _except_ Settings. All `$ wp option` commands are calling the `WP_Stream_Connector_Widgets` class instead of the `WP_Stream_Connector_Settings` class.

I happened to be using `$ wp option` for all of my testing and it just so happens that this was an isolated problem with one connector, not all of them. Making the premise of this issue incorrect. I will update the title.
